### PR TITLE
chore(docs):add Bader's info to openbeta-devs file

### PR DESCRIPTION
**Pull request by baderk**
---
name: Bader
about: walking through dev onboarding
title: ''
labels: ''
assignees: @actuallyyun 

---

## What type of PR is this?(check all applicable)
- [ ] refactor
- [ ] feature
- [ ] bug fix
- [ ] documentation
- [ ] optimization
- [x] other

## Description
walking through dev onboarding

### What this PR achieves
Adds my name as contributor.

### CODE REVIEW

File: src/assets/contributors-map.geojson [hunk 1]
```diff
@@ -4,15 +4,15 @@
     {
       "type": "Feature",
       "properties": {
-        "firstName": "Yun",
```
Comment (vnugent) [inline src/assets/contributors-map.geojson]: @baderk I think you need to add a new entry for your info, not updating the existing one (Yun)

**Review by vnugent — CHANGES_REQUESTED** [review]

File: src/assets/contributors-map.geojson (hunk 1 above)
Comment (baderk) [reply src/assets/contributors-map.geojson]: Oh I see yes I will change that

Comment (baderk) [reply src/assets/contributors-map.geojson]: Fixed the change here
3e08a7a94d65e11fd55c39bfc0b2bea91ef6079d

File: src/assets/contributors-map.geojson [hunk 2]
```diff
@@ -30,6 +30,21 @@
...
+      "type": "Feature",
+      "properties": {
+        "firstName": "Bader",
+        "githubId": "baderk",
+        "favoriteCrag": "becfb41f-3278-59c6-a376-2d23d97c220d"
+      },
+      "geometry": {
+        "type": "Point",
+        "coordinates": [
+          36.037803,
```
Comment (vnugent) [inline src/assets/contributors-map.geojson]: can you check if the coordinates are in [longitude, latitude] order?
Geojson specification: https://www.rfc-editor.org/rfc/rfc7946#section-3.1.1

If you can also use https://geojson.io/ to check the format of your file.  You should see drop pins if the format is correct.

**Review by vnugent — CHANGES_REQUESTED** [review]

File: src/assets/contributors-map.geojson (hunk 2 above)
Comment (baderk) [reply src/assets/contributors-map.geojson]: Thanks for pointing out the specification and the geojson tool. I just addressed it and I see the 3 points now on the map 👍🏼

**Review by vnugent — APPROVED** [review]

**Comment by vnugent** [comment]
@all-contributors add @baderk for code

**Pull request merged**